scoring/general-helpers.metta

```diff
@@ -0,0 +1,112 @@
+
+
+(= (addToSpace $row $space) (add-atom $space $row))
+
+(= (removeFromSpace $row $space) (remove-atom $space $row))
+
+(: foldr (-> (-> $a $b $b) $b $c $d))
+(= (foldr $f $i $xs)
+   (if (== $xs ())
+       $i
+       (let* (($h (car-atom $xs))
+              ($t (cdr-atom $xs))
+              ($ft (foldr $f $i $t)))
+         ($f $h $ft))))
+
+; replace expersion with Truth value from the space
+(= (replaceWithTruth $expr $space $spaceContent)
```

Comment:
I think this function is specific to the fitness function temporarily so it's best to move it there

Reply:
In fact. Except the `foldr` function, your newly added functions in this file should be moved to the `fitness.metta` file. If the functions defined here aren't being used by any other file than the `fitness.metta` file, its best to keep them in that file. For more details about this and other conventions we came up with you can refer the [Contributing File](https://github.com/iCog-Labs-Dev/metta-moses/blob/main/.github/CONTRIBUTING.md#code-conventions)